server/scsynth/SC_Lib_Cintf.h

```diff
@@ -43,7 +44,7 @@ extern Malloc gMalloc;
 extern HashTable<class SC_LibCmd, Malloc> *gCmdLib;
 
 void initialize_library(const char *mUGensPluginPath);
-void deinitialize_library();
+SCSYNTH_DLLEXPORT_C void deinitialize_library();
```

Comment:
then, maybe `World_Cleanup` should take a boolean argument to really clean up everything?

defining a public API function is a private header is bad nevertheless